### PR TITLE
Linux 6.6.18 eic7 x update

### PR DESCRIPTION
**Pull request by RevySR**

### CODE REVIEW

**Comment by NingYu00** [comment]
@RevySR 
Can you provide the reason for removing the "kvm_riscv_vcpu_update_config" function?

File: arch/riscv/errata/sifive/errata.c
Comment (linmineswincomputing) [inline arch/riscv/errata/sifive/errata.c:1]: It's ok to move the "riscv_cbom_block_size = 1....".  But why?

**Review by NingYu00 — APPROVED** [review]
Review +1

**Review by linmineswincomputing — APPROVED** [review]

**Review by JoeyZheng — APPROVED** [review]
+1

**Pull request merged**